src/main/java/com/google/cloud/tools/libraries/libraries.json

```diff
@@ -1,5 +1,105 @@
+    "clients": [
+      {
+      "name":"Google Java Client for Cloud Natural vision",
+      "vision":"java",
+      "site":"https://cloud.google.com/vision/docs/reference/libraries#client-libraries-install-java",
+      "apireference":"https://googlecloudplatform.github.io/google-cloud-java/latest/apidocs/com/google/cloud/vision/package-summary.html",
+      "infotip":"The com.google.cloud.vision.v1 packages",
+      "launchStage" : "beta",  
+      "packages" : ["com.google.cloud.vision.v1"],
+      "source" : "https://github.com/GoogleCloudPlatform/google-cloud-java/tree/master/google-cloud-vision",
+      "visionLevel" : "1.7.0",
+      "mavenCoordinates" : {
+         "groupId" : "com.google.cloud",
+         "artifactId" : "google-cloud-vision",
+         "version" : "0.22.0-beta"
+       }
+      }
+    ]
+  },
+  {
+    "name": "Google Cloud Natural Language",
+    "documentation": "https://cloud.google.com/language/docs/reference/libraries",
+    "description":"The Google Cloud Natural Language API provides natural language understanding technologies including sentiment analysis, entity analysis, and syntax analysis.",
+    "transports": ["grpc"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/svg/language.svg",
+    "clients": [
+      {
+      "name":"Google Java Client for Cloud Natural Language",
+      "language":"java",
+      "site":"https://cloud.google.com/language/docs/reference/libraries#client-libraries-install-java",
+      "apireference":"https://googlecloudplatform.github.io/google-cloud-java/latest/apidocs/com/google/cloud/language/package-summary.html",
+      "infotip":"The com.google.cloud.language.v1 packages",
+      "launchStage" : "beta",  
+      "packages" : ["com.google.cloud.language.v1"],
+      "source" : "https://github.com/GoogleCloudPlatform/google-cloud-java/tree/master/google-cloud-language",
+      "languageLevel" : "1.7.0",
+      "mavenCoordinates" : {
+         "groupId" : "com.google.cloud",
+         "artifactId" : "google-cloud-language",
+         "version" : "0.22.0-beta"
+       }
+      }
+    ]
+  },
+  {
+    "name": "Google Cloud Spanner",
+    "documentation": "https://cloud.google.com/spanner/docs/reference/libraries",
+    "description":"Cloud Spanner is a fully managed, relational database service built for transactional consistency, high availability, and global scale.",
+    "transports": ["grpc"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/svg/spanner.svg",
+    "clients": [
+      {
+      "name":"Google Java Client for Cloud Spanner",
+      "language":"java",
+      "site":"https://cloud.google.com/spanner/docs/reference/libraries#client-libraries-install-java",
+      "apireference":"https://googlecloudplatform.github.io/google-cloud-java/latest/apidocs/com/google/cloud/spanner/package-summary.html",
+      "infotip":"The com.google.cloud.spanner packages",
+      "launchStage" : "beta",  
```

Comment:
I personally find _launchStage_ a confusing keyword.

Reply:
I think we picked this at @paflynn's suggestion for compatibility with other docs.